docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
 
   18f-bot:
     build: .
-    image: 18f/charlie:58b784b1962fa2acad38e2a7161d9e98
+    image: 18f/charlie:ac42f36fc431e02d1ab1d352bc721af0
```

Comment:
Why are we publishing an image?

Reply:
In another project, we found that when we updated dependencies, they weren't getting reflected in our docker containers, so we started tagging our images based on the hash of the `package-log.json` file. Then when you do a `docker-compose up` after pulling down a change that includes changes to dependencies, that triggers a new build automatically. But it turns out it doesn't really work that great, either. You end up with a bunch of images over time that are no longer useful.

These aren't published anywhere, this is just a local tag. Still, I need to take this out because it's not as helpful as I thought it was at the time.